instrumentation/failsafe-3.0/library/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+plugins {
```

Comment:
could you also update https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md and add a readme similar to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/vibur-dbcp-11.0/library/README.md